roles/identity/defaults/main.yml

```
@@ -3,12 +3,12 @@
 identity_admin_username: admin
 identity_admin_password: null
 
-identity_keycloak_quarkus_version: "21.1.2"
+identity_keycloak_quarkus_version: "24.0.3"
```

```suggestion
identity_keycloak_quarkus_version: "24.0.5"
```

https://www.keycloak.org/2024/06/keycloak-2405-released.html